app/models/ems_refresh.rb

```diff
@@ -84,7 +84,7 @@ def self.get_target_objects(target, single_id = nil)
       target_class = target_class.to_s.constantize unless target_class.kind_of?(Class)
 
       if ManageIQ::Providers::Inventory.persister_class_for(target_class).blank? &&
-         [VmOrTemplate, Host, PhysicalServer, ExtManagementSystem, InventoryRefresh::Target].none? { |k| target_class <= k }
+         [VmOrTemplate, Host, PhysicalServer, PhysicalStorage, CloudVolume, ExtManagementSystem, InventoryRefresh::Target].none? { |k| target_class <= k }
```

Comment:
NOTE this is a bit of a strange legacy holdover from the old refresh code and we might want to delete it in the future.  I'm not sure we need this going forward since it is up to the individual provider to implement what types they support for targeted refresh.